### DISCUSSION TITLE
Virtual Training Day 등록 없이도 Power Platform study 가능한가요?

### DISCUSSION THREAD

**Discussion by gwh-21c** [fusiondevkr/pl900#49] [1 Q&A]
"퓨전 개발자 스터디" 과정을 최근에 알게 되어서 YOUTUBE를 통해서 스터디를 하고 있는 gwh입니다.
오늘 M365개발자 등록과 파워앱스 개발자과정에는 등록을 했는데 Virtual Training Day 신청은 이미 등록이 마감되었다고 합니다.
그래도 퓨전개발자 과정을 YOUTUBE로 스터디 하는데는 지장이 없는지요?

어떻게해서라도 기초과정을 따라가고 후속과정을 합류하고 싶은데 가능할지요?

**Comment by justinyoo** [comment]
@gwh-21c 안녕하세요! 우선 유튜브 동영상으로 준비하고 계시다고 하니 정말 반갑습니다!

파워 플랫폼 버추얼 트레이닝데이는 이미 8월 24일에 진행했던 부분이어서 유효하지 않은 행사구요, 이 스터디와는 전혀 상관 없습니다. 꼭 유튜브로 마무리 짓고 후속 과정에 신청해 주시면 좋겠습니다! 후속 과정에 대한 일정은 곧 알려드릴께요!
